stubs/CMakeLists.txt

```diff
@@ -14,6 +14,16 @@ ENDIF()
 
 PROJECT(@lib_name@)
 
+# check for and enable c++11 support
+INCLUDE(CheckCXXCompilerFlag)
+CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+IF(COMPILER_SUPPORTS_CXX11)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I don't believe that this is fully valid on Mac where the switch is `-stdlib=libc++`.  See what conda-forge does for more details: https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh